src/api.ts

```diff
@@ -42,9 +42,26 @@ export interface ErrorInfo {
   missingPlugin: string[];
 }
 
+/**
+ * All possible types of prop values.
+ *
+ * symbol is when prop value can't be represented (expression is used as prop value)
+ * undefined is used when when prop value can't be calculated
+ * null when prop is with {null} value
+ * */
+export type PropValue =
```

Comment:
I just realized this comment needs to be updated. I'll fix it tonight, then do a release.